products/omero/downloads/index.html

```diff
@@ -5,6 +5,7 @@
         <div class="callout large primary" id="bg-image-omero">
             <div class="row column text-center">
                 <h1>OMERO 5.3.2 Downloads</h1>
+                <a href="{{ site.baseurl }}/2017/05/16/omero-5-3-2.html" class="large btn-red button sites-button hide-for-small-only" style="text-shadow: none;">Read the Release Announcement</a>
```

Comment:
Since we are touching this section we may want to bump the version
5.3.3 release yesterday
When it becomes the main website, we will need to adjust the card on template board (note for myself!)